source/user-manual/registering/restful-api-registration.rst

```diff
@@ -0,0 +1,204 @@
+.. Copyright (C) 2019 Wazuh, Inc.
```

Comment:
A clarification about taking into account that the Wazuh API must be accesible from the Wazuh agent it will help.

Reply:
Done